data/Drabzit.json

```
@@ -1,7 +1,23 @@
 {
-  "name": "Drabzit",
-  "displayStatsPublic": true,
-  "type": "personal",
-  "bio": "Living 1 Life at a Time, Crazy IT Dude",
-  "avatar": "https://github.com/Drabzit.png"
+	"name": "Chris Bardsley",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Living 1 Life at a Time, Crazy IT Dude",
+	"avatar": "https://github.com/Drabzit.png",
+	"tags": ["Noob", "Gamer", "Dad"],
+
+
+	"socials": [{
+			"icon": "FaGithub",
+			"url": "https://github.com/Drabzit"
+		},
+		{
+			"icon": "FaLinkedin",
+			"url": "https:www.linkedin.com/in/christopher-bardsley-1291b0191"
+		},
+		{
+			"icon": "FaInstergram ",
```

Typo in the icon name.
```suggestion
			"icon": "FaInstagram ",
```

---

thanks - just found the page with all the icons - and updated it. before I saw this :D